bedrock/firefox/templates/firefox/new.html

```diff
@@ -123,15 +129,21 @@ <h1 class="large">
           </div>
           <div class="ios-links-wrapper latest-links-wrapper">
             <ul>
-              {% if l10n_has_tag('mobile_links') %}
-              <li>
-                <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">{{_('Get Firefox on your Android device')}}</a>
-              </li>
-              <li>
-                <a href="{{ url('firefox.os.index') }}">{{_('Learn about Firefox OS')}}</a>
-              </li>
+              {% if l10n_has_tag('ios_announce') %}
+                <li><a href="{{ url('newsletter.ios') }}">{{_('Learn more about Firefox on iOS')}}</a></li>
+              {% else %}
+                {% if l10n_has_tag('mobile_links') %}
```

Comment:
AFAIK. I considered trying to move the wrapper into the conditional logic because some locals could get a ul with no lis in it but decided to leave it the way I found it.